go.mod

```
@@ -9,7 +9,7 @@ require (
 	github.com/cosmos/cosmos-sdk v0.45.1
 	github.com/cosmos/go-bip39 v1.0.0
 	github.com/cosmos/ibc-go/v2 v2.2.0
-	github.com/ethereum/go-ethereum v1.10.17
+	github.com/ethereum/go-ethereum v1.10.16
```

```suggestion
	github.com/ethereum/go-ethereum v1.10.17
```
It should be updated to v1.10.17 already by #799.

---

The problem is this `go-ethereum` version breaks things with `btced` which leads to `go mod tidy` not working properly for systems that have a Go version prior to 1.17

---

I've opened #816 to make this clear